docs/developing-fuzzbench/adding_a_new_benchmark.md

```diff
@@ -35,22 +35,48 @@ powerful but also more work.
 
 ## OSS-Fuzz benchmarks
 
-You can use any existing OSS-Fuzz project as a benchmark. You just need to
-create a single file `oss-fuzz.yaml` in the benchmark directory.
+You can use most existing OSS-Fuzz projects a benchmark. First find out which
+project you want to use as a benchmark, and which commit and fuzz target you
+want to use. Then find out the date and time (UTC) of that commit and the path
```

Comment:
How to find date and time, why not find using hash, this feels hard.

Reply:
This would require building twice and is too much of a burden on fuzzbench IMO.

Reply:
Why cant be derive this by calling a git command on an oss-fuzz hash.

Reply:
Because the commit is from the project repo, not the OSS-Fuzz repo. AFAIK the OSS-Fuzz commit isn't given in revisions by CF: https://oss-fuzz.com/revisions?job=libfuzzer_ubsan_opencv&revision=201912170255

Actually, the date should be pretty easy to get since it is in the regression range.

Reply:
Also, the purpose of the date is so we can figure out the right version of the base-builder image to inherit from. If we get rid of date we might limit the number of benchmarks we can use since some benchmarks wont build with new clang versions. But the benefit is there will be 0 distinction between OSS-Fuzz and non-oss-fuzz benchmarks for the build system, everything can inherit from gcr.io/oss-fuzz/base-images/base-builder:latest. What do people think of that option?